exercises/perfect-numbers/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+pub enum Classification {
+

```

Comment:
I request that the three possible variants of `Classification` be included in the stub, since there is no point in keeping them from the student; the test expects them and the description lists them already.

The other reason is that the stub will not compile without it.

The precedent for this request is in https://github.com/exercism/rust/blob/master/exercises/forth/src/lib.rs#L7-L12 and https://github.com/exercism/rust/blob/master/exercises/robot-simulator/src/lib.rs#L4-L10.